docs/source/projections/cass.rst

```diff
@@ -1,10 +1,142 @@
 .. _cass:
 
 ********************************************************************************
-Cassini
+Cassini (Cassini-Soldner)
 ********************************************************************************
 
+Although the Cassini projection has been largely replaced by the Transverse Mercator, it is still in limited use outside the United States and was one of the major topographic mapping projections until the early 20th century.
+
++---------------------+-------------------------------------------------------------------------+
+| **Classification**  | Transverse and oblique cylindrical                                      |
++---------------------+-------------------------------------------------------------------------+
+| **Available forms** | Forward and inverse, Spherical and Elliptical                           |
++---------------------+-------------------------------------------------------------------------+
+| **Defined area**    | Global, but best used near the central meridian with long, narrow areas |
++---------------------+-------------------------------------------------------------------------+
+| **Implemented by**  | Gerald I. Evenden                                                       |
++---------------------+-------------------------------------------------------------------------+
+| **Options**                                                                                   |
++---------------------+-------------------------------------------------------------------------+
+| `+lat_0`            | Center of the map : latitude of origin                                  |
```

Comment:
I may be a big pedant, but I think it's more correct to keep it at "Latitude of origin" as the center is made up of both a latitude and a longitude.  Also, please add the default value of zero.